amy/consents/views.py

```diff
@@ -1,7 +1,14 @@
-from consents.forms import ActiveTermConsentsForm
+from consents.forms import ActiveTermConsentsForm, RequiredConsentsForm
 from consents.models import Consent
+from consents.util import person_has_consented_to_required_terms
+from django.contrib import messages
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.forms import HiddenInput
+from django.http.response import Http404
+from django.shortcuts import redirect, render
+from rest_framework.reverse import reverse
```

Comment:
This `isort` doesn't do great job here and in other places. It should have understood that `django` and `rest_framework` are our third party.

Reply:
@pbanaszkiewicz what do you use for ordering imports? How would you like it to run instead?

Reply:
I don't use anything automatic yet. I want to use isort, I'm just surprised that it worked that way.

https://www.python.org/dev/peps/pep-0008/#imports
> Imports should be grouped in the following order:
>
>   1. Standard library imports.
>   2. Related third party imports.
>   3. Local application/library specific imports.

Anyway, don't worry about it. I'll check out isort soon and see if it has some configuration problem or something.

Reply:
Oh I see. I think there's a configuration setting that can be modified. Before going down this route @pbanaszkiewicz can you tell me how you typically have handled imports?

Reply:
Opps looks like I asked my question as soon as you answered. no worries. Thanks for explaining